readme.md

```diff
@@ -141,6 +141,16 @@
 |  2 - 4  |    4     | 30-60m  |
 
 
+### [That's A Question!](https://boardgamegeek.com/boardgame/226322/s-question)
```

Comment:
In the realm of ridiculous, the [`awesome-lint` tool](https://npm.im/awesome-lint) will forever mark as a failure a heading ending in an exclamation point. I tried the HTML ASCII short code for the same character and it still is flagged. What seems to work, and what I'm suggesting, is to actually use the unicode emoji character for an exclamation point.

```suggestion
### [That's A Question﹗](https://boardgamegeek.com/boardgame/226322/s-question)
```